Automattic/wp-cldr
# Update CLDR version to 41

This PR is based on #95, updating the CLDR version to 41. 
Additionally, I've updated the list of locales, for both WPCOM and WordPress.org. 

Since version 36, CLDR-JSON was moved to https://github.com/unicode-org/cldr-json, with all data stored in 1 repository. 
I've extracted the WordPress language list from `https://api.wordpress.org/translations/core/1.0/ ` 

### Testing instructions

1. Run `./get-cldr-files.sh`
2. Confirm that a new 41.0.0 folder was created in the `data` folder
3. Run `php prune-cldr-files.php` to delete unneeded files. 
4. Confirm that the size of the `41.0.0` folder is similar to the already existing `29.0.0`

Once the PR is approved, I'll follow up with a PR to also commit the data.
